src/components/Button.tsx

```diff
@@ -0,0 +1,13 @@
+import { type ButtonHTMLAttributes } from 'react';
+
+interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {}
+
+const Button: React.FC<ButtonProps> = ({ className, children, ...props }) => {
+  return (
+    <button className={`${className}`} {...props} type="button">
```

Comment:
className 만 특별히 string 으로 변환해서 주입해주는 이유가 있을까요? {...props} 로 적용해도  같은 동작을 할 것 같다고 예상되서요!

```suggestion
    <button  {...props} type="button" >
```

Reply:
button을 사용하는 쪽 props에 className이 들어오면 className이 덮어씌워져서 css가 적용이 안되는 문제 때문에 템플릿 리터럴로 적용했습니다!